src/test/java/htsjdk/samtools/ValidateSamFileTest.java

```diff
@@ -552,12 +552,6 @@ public void duplicateReadsOutOfOrder() throws Exception {
                         "@RG\tID:0\tSM:Hi,Mom!\n" +
                         "E\t147\tchr1\t15\t255\t10M\t=\t2\t-30\tCAACAGAAGC\t)'.*.+2,))\tU2:Z:CAA";
 
-        final String SOTagCorrectlyProcessTestData =
```

Comment:
why is this removed?

Reply:
thank you for your reply. We've removed this test case since as @a-filipp0v said, it's redundant after our implementation.

Reply:
not sure I understand...if it's just "Redundant" but not exactly equals, I'd ask that it be left in.

Reply:
This case does not pass now, since we no longer treat SO:NOTKNOWN as error as test expects.

Reply:
got it